src/app/routes/regex/index.test.js

```diff
@@ -1,16 +1,16 @@
 import { matchPath } from 'react-router-dom';
 import {
-  articleRegexPath,
-  articleDataRegexPath,
-  articleSwRegexPath,
-  articleManifestRegexPath,
-  frontpageRegexPath,
-  frontpageDataRegexPath,
-  frontpageManifestRegexPath,
-  frontpageSwRegexPath,
-  cpsAssetPageRegexPath,
-  cpsAssetPageDataRegexPath,
-  radioAndTvRegexPathsArray,
+  articlePath,
+  articleDataPath,
+  articleSwPath,
+  articleManifestPath,
+  frontPagePath,
+  frontPageDataPath,
+  frontPageManifestPath,
+  frontPageSwPath,
+  cpsAssetPagePath,
+  cpsAssetPageDataPath,
+  radioAndTvPath,
```

Comment:
There isn't a `radioAndTvDataPath` test, could you open an issue to add a test for this please.